packages/Babylonian-Core.package/BPExample.class/instance/traceInformation.st

```diff
@@ -0,0 +1,4 @@
+accessing
```

Comment:
Can we do without this method? The traceInformation is not to be shared with anyone. This is only a private structure taking care of preserving the consistency between the trace object and the tracing process.